MetaData/python/JobConfig.py

```diff
@@ -224,6 +225,13 @@ def parse(self):
             self.readProcessIdMap(self.options.processIdMap)
         
         if self.useAAA:
+            import subprocess
+            tleft = int(subprocess.check_output(["voms-proxy-info", "--actimeleft"]))
```

Comment:
This code is normally run by cmsRun when it loads the configuration and it may be better not to do it for each job.

I would probably execute the command only if the